phpmailer/phpmailer.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Name: PHP Mailer SMTP
+ * Description: Connects to a SMTP server based on the config
+ * Version: 0.1
+ * Author: Marcus Mueller <http://mat.exon.name>
+ */
+
+use Friendica\App;
+use Friendica\Core\Addon;
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\Exception;
+
+function phpmailer_install()
+{
+	Addon::registerHook(
+		'emailer_send_prepare',
+		'addon/phpmailer/phpmailer.php',
+		'phpmailer_emailer_send_prepare'
+	);
+}
+
+function phpmailer_uninstall()
+{
+	Addon::unregisterHook(
+		'emailer_send_prepare',
+		'addon/phpmailer/phpmailer.php',
+		'phpmailer_emailer_send_prepare'
+	);
+}
+
+function phpmailer_module()
```

Comment:
This addon doesn't provide a module (a specific page in the Friendica interface), please remove this function.

Reply:
Function removed.